modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/quantity_selector/QuantitySelector.js

```diff
@@ -266,38 +126,40 @@ function QuantitySelector(props) {
 	);
 }
 
+QuantitySelector.defaultProps = {
+	disabled: false,
+	inputSize: 'default',
+	settings: {
+		allowedQuantity: [],
+		maxQuantity: 99,
+		minQuantity: 1,
+		multipleQuantity: 1,
+	},
+	style: 'select',
+};
+
 QuantitySelector.propTypes = {
-	allowedQuantities: PropTypes.arrayOf(PropTypes.number),
 	appendedIcon: PropTypes.string,
 	appendedText: PropTypes.string,
+	disableAddToCartButton: PropTypes.bool,
+	disableQuantitySelector: PropTypes.bool,
 	disabled: PropTypes.bool,
 	inputName: PropTypes.string,
-	maxQuantity: PropTypes.number,
-	minQuantity: PropTypes.number,
-	multipleQuantity: PropTypes.number,
-
-	/**
-	 * if 'throttleOnUpdate' is true,
-	 * 'onUpdate' must return a <Promise>.
-	 */
-	onUpdate: PropTypes.func,
+	inputSize: PropTypes.string,
 	prependedIcon: PropTypes.string,
 	prependedText: PropTypes.string,
 	quantity: PropTypes.number,
+	rtl: PropTypes.bool,
```

Comment:
I'm not aware of RTL enabling at the level of the code (excluding CSS or styling). RTL should either be driven by LFR Portal (Language selector portlet, for instance) which alters the `dir` attribute, or from the other/browser scope anyway. Let's check this together. :)